lib/Folder/FolderManager.php

```diff
@@ -110,29 +110,48 @@ public function getAllFoldersWithSize($rootStorageId) {
 
 		$rows = $query->execute()->fetchAll();
 
+		$folderMappings = $this->getAllFolderMappings();
+
 		$folderMap = [];
 		foreach ($rows as $row) {
 			$id = (int)$row['folder_id'];
+			$mappings = $folderMappings[$id];
```

Comment:
```suggestion
			$mappings = $folderMappings[$id] ?? [];
```

to handle the case where there is no mapping for the folder